prime-router/docs/playbooks/dependabot-updates.md

```diff
@@ -0,0 +1,56 @@
+# How to do Dependabot Updates
+
+## Introduction
+As part of using GitHub, we have the dependabot service which automatically finds updates
+for project dependencies and creates PRs for them. While this is really helpful, if you're
+not careful with them, you can really mess up the application. Spider-Man rules apply here:
+"With great power comes great responsibility"
+
+Below are the steps for how to most safely check and merge in the dependabot PRs:
+
+## Steps
+1. Checkout master and get latest:
+```shell
+git checkout master
+git fetch --all --prune
+git pull --ff-only
+```
+2. Create a new local branch to work with based on master. This branch is never going back up to GitHub so the name you choose is up to you:
```

Comment:
I don't follow why create a branch to merge the dependabot branch.  I normally use the same dependabot branch to build, and if I need to merge in master I just do a @ dependabot rebase.  Is there another function to this extra branch?